.github/workflows/test.yml

```diff
@@ -3,18 +3,6 @@ name: Tests
 on: [pull_request]
 
 jobs:
-  typescript:
```

Comment:
Just gonna let vitest verify types yeah?

Reply:
yeah, I put it in the desc for the PR, this is what vitest recommends

> this PR also removed the script and github workflow to run tsc --no-emit per the recommendation by vitest at the bottom of [this page](https://vitest.dev/guide/testing-types#testing-types)

Reply:
Cool. What about this? Do we even need that package? 
> Vitest uses tsc --noEmit or vue-tsc --noEmit, depending on your configuration, so you can remove these scripts from your pipeline.

Reply:
I don't follow, what package are you referring to?

Reply:
Sorry, vue-tsc. Seems like vitest already had that as a dependency? Idk just wondering

Reply:
vitest is setup to use it, but I don't know that it will install for you. We will need it for our build script too

Reply:
Remind me why we need it for the build?

Reply:
The build before this PR called `tsc && vite build`. Calling tsc will error on .vue import.

Reply:
Is that for type checking or for building types? Because we also have vite-dts which bundles the types for dist.

Reply:
Type checking for sure, not sure about building.

Reply:
Wonder if we even need it. We already test the types are valid with vitest. Maybe we don't need that as part of the build step. 

Approved either way.

Reply:
I don't think vitest can properly test types if vue files are in the mix.